src/main/java/org/osc/sdk/manager/element/ManagerSecurityGroupInterfaceElement.java

```diff
@@ -34,10 +36,21 @@
      */
     String getName();
 
-    /**
-     * @return the identifier of this policy defined by the security manager
-     */
-    String getPolicyId();
+	/**
+	 * Provide manager security group id if the manager is supporting policy
```

Comment:
Provides the identifier of the security group defined by security managers that support policy mapping and security groups. 
Also please follow a similar text for other places